L1Trigger/L1THGCal/src/be_algorithms/HGCalHardwareInterface.cc

```diff
@@ -0,0 +1,205 @@
+#include "L1Trigger/L1THGCal/interface/be_algorithms/HGCalHardwareInterface.h"
+
+
+bool HGCalHardwareInterface::DecodeClusterData (const ClusterDataContainer& DataToBeDecoded){
+  if(DataToBeDecoded.empty()) 
+    return false;
+  HGCMC_->SetClusterET(            get32bitPart_(DataToBeDecoded.at(ClusterDataFormat::ET/wordsize_),             
```

Comment:
The initial idea (as described above) to be able to decode a cluster within the Multicluster object. I am reverting this idea until I get a feedback.